src/Elmish/Enzyme/Foreign.purs

```diff
@@ -112,6 +114,18 @@ find selector w = liftEffect $ runEffectFn2 find_ selector w
 parent :: forall m n. MonadEffect m => Wrapper n -> m (Wrapper n)
 parent w = liftEffect $ runEffectFn1 parent_ w
 
+-- | Returns the child nodes of the given `Wrapper`. See
+-- | https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/children.html ffor
+-- | more info.
+children :: forall m n. MonadEffect m => Wrapper n -> m (Wrapper ManyNodes)
+children w = liftEffect $ runEffectFn1 children_ w
+
+-- | Returns the child node at index `idx` of the given `Wrapper`. See
+-- | https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/childAt.html ffor
+-- | more info.
+childAt :: forall m n. MonadEffect m => Int -> Wrapper n -> m (Wrapper SingleNode)
```

Comment:
Is it not the same as `children >> at n`?

Reply:
Yeah, it should be. Just a convenience function. Do you think it’s better to get rid of this and just use children?

Reply:
Not sure. This would become evident with practical use.

So I think I'd leave it in for now, but add a comment explaining this.